.github/workflows/aggregation.yml

```diff
@@ -5,7 +5,7 @@ on:
   workflow_dispatch:
   push:
     paths:
-      - "aggregation/"
+      - "aggregation/**"
```

Comment:
unrelated fix: without this glob pattern, the CI pipeline is not triggered as expected.